instrumentation/hibernate/hibernate-6.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/hibernate/v6_0/AbstractHibernateTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.hibernate.v6_0;
+
+import io.opentelemetry.instrumentation.testing.junit.AgentInstrumentationExtension;
+import io.opentelemetry.instrumentation.testing.junit.InstrumentationExtension;
+import java.util.ArrayList;
+import java.util.List;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+@SuppressWarnings("deprecation") // testing instrumentation of deprecated class
+public abstract class AbstractHibernateTest {
+  protected static SessionFactory sessionFactory;
+  protected static List<Value> prepopulated;
+
+  @RegisterExtension
+  protected static final InstrumentationExtension testing = AgentInstrumentationExtension.create();
+
+  @BeforeAll
+  protected static void setup() {
+    sessionFactory = new Configuration().configure().buildSessionFactory();
+    // Pre-populate the DB, so delete/update can be tested.
+    Session writer = sessionFactory.openSession();
+    writer.beginTransaction();
+    prepopulated = new ArrayList<Value>();
+    for (int i = 0; i < 5; i++) {
+      prepopulated.add(new Value("Hello :) " + i));
+      writer.save(prepopulated.get(i));
```

Comment:
would using `persist` here instead of `save` get rid of the deprecation warning?

Reply:
Done. Thanks.